src/utils/convertLocalTime.ts

```diff
@@ -0,0 +1,36 @@
+const convertLocateTime = (date: Date) => {
+    return new Date(date.getTime() - date.getTimezoneOffset() * 60 * 1000);
+};
+
+interface convertDateTimeProps {
+    time: string;
+    options?: {
+        showDate?: boolean;
+        showTime?: boolean;
+    };
+}
+const convertDateTime = (props: convertDateTimeProps): string => {
+    const { time, options } = props;
+
+    const timeStamp = convertLocateTime(new Date(time));
+
+    if (options) {
+        if (!options.showDate && options.showTime) {
+            return timeStamp.toLocaleString("en-US", { hour: "2-digit", minute: "2-digit" });

```

Comment:
Why is this en-US?

Reply:
Because I want the time to show AM/PM and en-US allow timeStamp to show it. For some language code such as vi(Vietnamese) and en-GB (British English) doesn't have AM/PM. Moreover, I think that 03:00 PM is more easily to understand than 15:00.

Reply:
Let's just leave the time problem to solve later. I think we should delete this file